File: Android-Java/app/src/main/java/io/supernova/spacebook/adapter/TwoActivityPagerAdapter.java
/**
 *  Created by Supernova.
 */

package io.supernova.spacebook.adapter;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;
import java.util.*;


public class TwoActivityPagerAdapter extends FragmentPagerAdapter {
    public final List<Fragment> fragments;
    
    public TwoActivityPagerAdapter(FragmentManager fragmentManager, List<Fragment> fragments) {
        super(fragmentManager);
        this.fragments = fragments;
    }
    @Override
    public Fragment getItem(int position) {
    
        return fragments.get(position);
    }
    @Override
    public int getCount() {
    
        return fragments.size();
    }
}

File: Android-Kotlin/app/src/main/java/io/supernova/spacebook/adapter/TwoActivityPagerAdapter.kt
/**
 * Created by Supernova.
 */

package io.supernova.spacebook.adapter

import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentPagerAdapter
import java.util.*


class TwoActivityPagerAdapter(fragmentManager: FragmentManager, fragments: List<Fragment>): FragmentPagerAdapter(fragmentManager) {

    val fragments: List<Fragment>
    
    init {
        this.fragments = fragments
    }
    override fun getItem(position: Int): Fragment {
    
        return fragments.get(position)
    }
    override fun getCount(): Int {
    
        return fragments.size
    }
}




